beard/metrics/text.py

```diff
@@ -162,3 +164,51 @@ def jaro_winkler(s1, s2, p=0.1):
             break
 
     return jaro_distance + p * common_prefix * (1 - jaro_distance)
+
+
+def levenshtein(a, b):
+    """Calculate the levenshtein distance between strings a and b.
+
+    Case sensitiveness is activated, meaning that uppercase letters
```

Comment:
May be we would consider using this package for Levenshtein dissimilarity: https://pypi.python.org/pypi/python-Levenshtein/0.12.0 or at least check if ours is faster